app/Http/Controllers/Assets/AssetCheckinController.php

```diff
@@ -110,6 +110,13 @@ public function store(AssetCheckinRequest $request, $assetId = null, $backto = n
             $checkin_at = $request->input('checkin_at');
         }
 
+        if(!empty($asset->licenseseats->all())){
+            foreach ($asset->licenseseats as $seat){
+                $seat->assigned_to = null;
```

Comment:
I'm not sure why we need this here? License checkins would never be working if we didn't elsewhere set their assigned to to `null`.